addons/viewport/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@storybook/addon-viewport",
+  "version": "3.2.0",
+  "description": "Storybook addon to change the viewport size to mobile",
+  "main": "dist/index.js",
```

Comment:
This file is broken

Reply:
#2589 fixes that by removing the `index.js` file and using `register.js` (or should it be `manager.js`?) as `main`